[2021-12-15 16:25:01,702] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-15T16:24:57.805303+00:00 [queued]>
[2021-12-15 16:25:01,726] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-15T16:24:57.805303+00:00 [queued]>
[2021-12-15 16:25:01,727] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-15 16:25:01,728] {taskinstance.py:1242} INFO - Starting attempt 1 of 3
[2021-12-15 16:25:01,731] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-15 16:25:01,771] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): check_imageFile> on 2021-12-15 16:24:57.805303+00:00
[2021-12-15 16:25:01,784] {standard_task_runner.py:52} INFO - Started process 237 to run task
[2021-12-15 16:25:01,792] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'check_imageFile', 'manual__2021-12-15T16:24:57.805303+00:00', '--job-id', '1401', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpcbzy5uks', '--error-file', '/tmp/tmpxvt7ueor']
[2021-12-15 16:25:01,794] {standard_task_runner.py:77} INFO - Job 1401: Subtask check_imageFile
[2021-12-15 16:25:01,918] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-15T16:24:57.805303+00:00 [running]> on host 218df5ddc363
[2021-12-15 16:25:02,087] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=check_imageFile
AIRFLOW_CTX_EXECUTION_DATE=2021-12-15T16:24:57.805303+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-15T16:24:57.805303+00:00
[2021-12-15 16:25:02,095] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-15 16:25:02,098] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sha256sum /opt/***/official_data/Image.csv']
[2021-12-15 16:25:02,125] {subprocess.py:85} INFO - Output:
[2021-12-15 16:25:02,138] {subprocess.py:89} INFO - e891870a24e5a22c9520c5d417af923366083a3617288cafa3cc18e3fed77b11  /opt/***/official_data/Image.csv
[2021-12-15 16:25:02,139] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-15 16:25:02,191] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=check_imageFile, execution_date=20211215T162457, start_date=20211215T162501, end_date=20211215T162502
[2021-12-15 16:25:02,248] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-15 16:25:02,411] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
